### PR TITLE
Updated full name field backend and frontend

### LINKED ISSUE #71: Backend for saving user information from feide
- [x] POST, UPDATE, GET, DELETE functionality for saving user information
- [x] Create proper columns in User table

### PR DESCRIPTION
**Pull request by PeterRydberg**
Visual update for name from Feide with updated backend tests 
Resolves #71

### CODE REVIEW

File: frontend/src/components/Header/Presentational.tsx [hunk 1]
```diff
@@ -21,17 +20,7 @@ export const Presentational: React.FunctionComponent<IProps> = (props) => {
...
-        <KnappBase
-          type="hoved"
-          htmlType="button"
-          id="logoutButton"
-          onClick={onClickAction}
-        >
-          Admin
-        </KnappBase>
-      </Link>
+      You're logged in as {userInformation.fullname}
```
Comment (s0lvang) [inline frontend/src/components/Header/Presentational.tsx]: Why is it red?

Comment (Adrianht) [reply frontend/src/components/Header/Presentational.tsx]: > Why is it red?

I think it' s just a bug on Github, it happens sometimes.

Comment (FluidSense) [reply frontend/src/components/Header/Presentational.tsx]: The red background-color is definitely being caused by the error-highlighting feature of GitHub's text editor/viewer. You can find the same behaviour occurring to an intended block comment in another JSON file on GitHub:

Comment (FluidSense) [reply frontend/src/components/Header/Presentational.tsx]: I think the correct syntax would be to wrap the entire thing in a formatted string, like:
`{'You're logged in as ${userinformation.fullname}'`

Comment (FluidSense) [reply frontend/src/components/Header/Presentational.tsx]:
```suggestion
      {`You're logged in as ${userInformation.fullname}`}
```

**Pull request merged**